docs/CHANGELOG.asciidoc

```diff
@@ -56,6 +57,264 @@ Review important information about the {kib} 8.x releases.
+Machine Learning::
+* Adds E5 model configurations ({kibana-pull}172053[#172053]).
+* Adds the ability to create a categorization anomaly detection job from pattern analysis ({kibana-pull}170567[#170567]).
+* Adds and displays alerts data in the Anomaly Explorer ({kibana-pull}167998[#167998]).
+Observability:: 
+* Adds logic to update flyout highlights ({kibana-pull}172193[#172193]).
+* Adds logic to display highlights in the flyout ({kibana-pull}170650[#170650]).
+* Changes the Custom threshold title to Beta ({kibana-pull}172360[#172360]).
+Security:: 
+* Disables the connector parameters field ({kibana-pull}173610[#173610]).
+* Adds a risk engine missing privileges callout ({kibana-pull}171250[#171250]).
+* Asset criticality privileges API ({kibana-pull}172441[#172441]).
+Uptime::
+* Global params Public APIs ({kibana-pull}169669[#169669]).
+* Private location public API's ({kibana-pull}169376[#169376]).
+* Settings public API ({kibana-pull}163400[#163400]).
+
+For more information about the features introduced in 8.12.0, refer to <<whats-new,What's new in 8.12>>.
+
+[[enhancements-and-bug-fixes-v8.12.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.12.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.12.0]]
+=== Enhancements
+Alerting::
+* Auto close ServiceNow incidents when alerts are resolved ({kibana-pull}171760[#171760]).
+* PagerDuty connector now supports the links and `custom_details` attributes ({kibana-pull}171748[#171748]).
+* Adds a mute and unmute action component in the alerts table row actions ({kibana-pull}170651[#170651]).
+* Extends the PagerDuty connector API to support the `links` and `custom_details` attributes provided by the Event API ({kibana-pull}170459[#170459]).
+* Adds toggle for alert as data fields in alert templating ({kibana-pull}170162[#170162]).
+APM::
+* Perform functions and LLM interactions on the server ({kibana-pull}172590[#172590]).
+* Adds viewInApp URL to the custom threshold rule type ({kibana-pull}171985[#171985]).
+* Adds the KQL bar to embeddables ({kibana-pull}171016[#171016]).
+* Enables the average mobile app launch time panel ({kibana-pull}170773[#170773]).
+* Enables the mobile most launches panel ({kibana-pull}168925[#168925]).
+* Improves the Differential TopN functions grid view ({kibana-pull}170008[#170008]).
+Cases::
+* Users can copy to the clipboard the hashes of files uploaded to cases ({kibana-pull}172450[#172450]).
+* Allow users to configure which columns are displayed in the cases list including custom fields ({kibana-pull}170950[#170950]).
+* Adds a new sub-feature privilege to control user access to the cases settings ({kibana-pull}170635[#170635]).
+Dashboard::
+* Adds Links to the Visualization library ({kibana-pull}170810[#170810]).
+Discover::
+* Adds a field tokens column in the grid header ({kibana-pull}167179[#167179]).
+* Enables the addition of columns from the document viewer ({kibana-pull}171083[#171083]).
```

Comment:
```suggestion
* Enables the addition of columns from the document viewer when using ES|QL ({kibana-pull}171083[#171083]).
```
Because when not using ES|QL this was already possible